### PR TITLE
[tests-only] [full-ci] Added API test to search shared files by sharee using tag

### PR DESCRIPTION
**Pull request by nabim777**
## Description
This PR adds the API tests for full search api. The scenarios added in this PR are
- search file using tag by sharee

## Related Issue
- https://github.com/owncloud/ocis/issues/6606#event-9635806756

## Motivation and Context
- there was no test coverage for api test for search file using tag by sharee. so, this PR covers the require scenario test case

## How Has This Been Tested?
- test environment:
- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by SwikritiT — CHANGES_REQUESTED** [review]
Don't ask reviews in other dependent PR's before the first PR is merged https://github.com/owncloud/ocis/pull/6641

File: tests/acceptance/features/apiGraph/fullSearch.feature [hunk 1]
```diff
@@ -27,3 +27,27 @@ Feature: full text search
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario Outline: as a sharee search a file tagged by sharer using tag
```
Comment (SwikritiT) [inline tests/acceptance/features/apiGraph/fullSearch.feature:32]:
```suggestion
  Scenario Outline: sharee searches a file tagged by sharer using tag
```

File: tests/acceptance/features/apiGraph/fullSearch.feature [hunk 2]
```diff
@@ -27,3 +27,27 @@ Feature: full text search
...
+    And user "Brian" has accepted share "/uploadFolder" offered by user "Alice"
+    When user "Brian" searches for "Tags:tag1" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result of user "Brian" should contain only these files:
+      | file1.txt |
+      | file2.txt |
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
```
Comment (SwikritiT) [inline tests/acceptance/features/apiGraph/fullSearch.feature:53]: what about spaces?

Comment (nabim777) [reply tests/acceptance/features/apiGraph/fullSearch.feature:53]: There is a issue in relate to space https://github.com/owncloud/ocis/issues/6669

**Review by PrajwolAmatya — COMMENTED** [review]
lgtm

**Comment by SwikritiT** [comment]
@nabim777 the branch is 14 commits behind master
https://github.com/owncloud/ocis/tree/test-api-search-tag-by-user
please rebase and push

**Review by SwikritiT — COMMENTED** [review]
the changes LGTM

**Review by SwikritiT — APPROVED** [review]
LGTM

**Review by saw-jan — APPROVED** [review]
LGTM

**Pull request merged**